integrationtests/self/datagram_test.go

```diff
@@ -73,7 +74,7 @@ var _ = Describe("Datagram test", func() {
 							return false
 						}
 						// don't drop Long Header packets
-						if packet[0]&0x80 == 1 {
```

Comment:
Fwiw, the old version here was incorrect.

Reply:
I don't think so, this is selecting long header packets?

Reply:
Now I see it. It would have to be `&0x80 == 0x80`. Nice one.